Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -165,19 +158,22 @@ public function reset(): void
         $this->checkpointStorage->acquireLock();
         $this->checkpointStorage->updateAndReleaseLock(SequenceNumber::none());
 
-        $contentGraph = $this->getState();
-        foreach ($contentGraph->getSubgraphs() as $subgraph) {
-            $subgraph->inMemoryCache->enable();
+        $contentGraphFinder = $this->getState();
+        /** @var ContentGraph $contentGraph */
+        foreach ($contentGraphFinder->getInstances() as $contentGraph) {
+            foreach ($contentGraph->getSubgraphs() as $subgraph) {
+                $subgraph->inMemoryCache->enable();
+            }
```

Comment:
Not entirely new, but this code just made me a bit suspicious and I wonder if it is still needed when we switch to synchronous catchups